site/dat/repo/various.json

```diff
@@ -1534,12 +1534,12 @@
       }
     }
   },
-  {
+   {
     "id": "jmeter-atakama-variabilization-plugin",
     "name": "Atakama Variabilization Plugin",
-    "description": "Variabilization plugin for jmeter",
-    "screenshotUrl": "",
-    "helpUrl": "https://github.com/atakama/jmeter_plugin_variabilization",
+    "description": "Variabilization plugin for jmeter: https://github.com/atakama/jmeter_plugin_variabilization",
+    "screenshotUrl": "https://user-images.githubusercontent.com/2484866/182579087-59225ac5-8abb-4a87-835a-2f202b1374fd.png",
+    "helpUrl": "https://ph-eval.atakama-technologies.com/downloadWebsite/index.php/product/show?id=1",
```

Comment:
This is not help page, it's a promotional landing page. Let's not confuse users of plugins.

Reply:
There is a video link  with how it works also. I can put it within the description?
What do you suggest?

Reply:
We can also add a documentation link within this promotional landing page

Reply:
I think it would be right to put your landing page into description, where users supposed to get general information about the plugin. The help link should serve exactly the technical help page.